packages/middleware-ssec/src/index.ts

```diff
@@ -78,3 +77,19 @@ export const getSsecPlugin = (config: PreviouslyResolved): Pluggable<any, any> =
     clientStack.add(ssecMiddleware(config), ssecMiddlewareOptions);
   },
 });
+
+function isValidBase64(str: string) {
```

Comment:
This function should be named after its logical behavior. It is not checking base64 validity.

It is checking whether the input is a base64 encoded SSE customer key.